headers/storagemanager.h

```diff
@@ -100,6 +101,13 @@ struct AddonOptions {
 	int i2cAnalog1219Block;
 	uint32_t i2cAnalog1219Speed;
 	uint8_t i2cAnalog1219Address;
+	uint8_t gpDaughterSDAPin;
```

Comment:
Options that will persist to EEPROM need to stay in order to avoid breaking saved configs on previous versions. These new fields should go before the `checksum` field.

Reply:
Done